games/brickmatch/Kconfig

```diff
@@ -33,6 +33,39 @@ config GAMES_BRICKMATCH_STACKSIZE
 	int "Brickmatch Game stack size"
 	default DEFAULT_TASK_STACKSIZE
 
+#
+# Output Device Selection
+#
+
+choice
+	prompt "Output Device (LED Matrix, SPI Screen, etc)"
+	default GAMES_BRICKMATCH_USE_SPI_SCREEN
```

Comment:
Actually it should be GAMES_BRICKMATCH_USE_LCD_SCREEN because by default it works with any LCD screen, not only APA103 MATRIX as LCD